cmd/tailscale/ui.go

```diff
@@ -560,6 +560,12 @@ func (ui *UI) layout(gtx layout.Context, sysIns system.Insets, state *clientStat
 							ui.showCopied(gtx, a)
 						}
 					}
+
+					if p.Peer.Hostinfo.Location() != nil {
```

Comment:
Would we ever set location for some reason other than it being a location based exit node?

Reply:
Not currently, no - it is only implemented for one specific type of node currently.

Reply:
We should either guarantee that it will never be used for anything else, or add a comment to change the filtering in the clients if we ever do

Reply:
Added the isWireguardOnly check, that should give us some room to breathe